capstone/config.py

```diff
@@ -7,11 +7,13 @@
 
 capstone_api_token = os.getenv("CAPSTONE_API_TOKEN", "test123")
 
-gitto_base_url = os.getenv("CAPSTONE_GITTO_BASE_URL", "http://localhost:8080")
+gitto_base_url = os.getenv("CAPSTONE_GITTO_BASE_URL", "http://localhost:7878")
 gitto_api_token = os.getenv("CAPSTONE_GITTO_API_TOKEN", "gitto")
 
 data_dir = os.getenv("CAPSTONE_DATA_DIR", "data")
 
+docker_registry = os.getenv("CAPSTONE_DOCKER_REGISTRY", "registry.local.pipal.in:8080")
```

Comment:
This shouldn't run on port 8080.

Reply:
why? It is running in Nomad. Nginx in Nomad is running on port 8080.